net/pbr/Makefile

```
@@ -5,56 +5,42 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=pbr
 PKG_VERSION:=1.2.1
-PKG_RELEASE:=41
+PKG_RELEASE:=45
 PKG_LICENSE:=AGPL-3.0-or-later
 PKG_MAINTAINER:=Stan Grishin <stangri@melmac.ca>
 
 include $(INCLUDE_DIR)/package.mk
 
-define Package/pbr/default
-  SECTION:=net
-  CATEGORY:=Network
-  SUBMENU:=Routing and Redirection
-  TITLE:=Policy Based Routing Service
-  URL:=https://github.com/stangri/pbr/
-  DEPENDS:=+ip-full +jshn +jsonfilter +resolveip
+define Package/pbr
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Routing and Redirection
+	TITLE:=Policy Based Routing Service with nft/nft set support
+	URL:=https://github.com/stangri/pbr/
+	DEPENDS:=+ip-full +jshn +jsonfilter +resolveip
```

Why doing this in such complicated way?

Something like this should work for you
```suggestion
	DEPENDS:= \
+ip-full \
+jshn \
+jsonfilter \
+resolveip \
+!BUSYBOX_DEFAULT_AWK:gawk \
+!BUSYBOX_DEFAULT_GREP:grep \
+!BUSYBOX_DEFAULT_SED:sed \
+kmod-nft-core \
+kmod-nft-nat \
+nftables-json
```